### PR TITLE
feat: Add type of asset bump in the IndexesUpdateAssetHistory entity

### LINKED ISSUE #16: Add the type of Asset Nonce action (accept or cancel listing)
We need to know if the bump was caused by a cancel action of the fact of a tenant accepting a listing so the cancel job ignores the bump.

### PR DESCRIPTION
**Pull request by juanmahidalgo**
Closes #16

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -31,6 +31,11 @@ enum IndexUpdateType @entity {
   ASSET
 }
 
+enum IndexUpdateEventType @entity {
```
Comment (fzavalia) [inline schema.graphql:34]: You can remove the @entity on an enum I believe

File: src/handlers/rentals.ts [hunk 2]
```diff
@@ -55,6 +56,13 @@ export function handleAssetRented(event: AssetRented): void {
     currentRental.save()
   }
 
+  let assetNonceUpdateHistoryIdDueToRent = getIndexesUpdatesHistoryNextCount().value.minus(BigInt.fromI32(1))
```
Comment (fzavalia) [inline src/handlers/rentals.ts:59]: You should do this for the last 2 asset index updates, not just the last one

**Review by fzavalia — APPROVED** [review]

File: src/handlers/rentals.ts [hunk 3]
```diff
@@ -55,6 +56,17 @@ export function handleAssetRented(event: AssetRented): void {
     currentRental.save()
   }
 
+  let lastAssetNonceUpdateHistoryId = getIndexesUpdatesHistoryNextCount().value.minus(BigInt.fromI32(1))
```
Comment (LautaroPetaccio) [inline src/handlers/rentals.ts:59]: `getAssetIndexesUpdatesHistoryCount` is the function that you're looking for here.

Comment (LautaroPetaccio) [reply src/handlers/rentals.ts:59]: `getAssetIndexesUpdatesHistoryCount` returns the index of the next `AssetNonceUpdateHistory`. Taking into consideration that the event to increase the nonce has already been processed, the current index is the one that you would like to set. The previous indexes might not be relevant or might not exist as the index 0 is not generated through an event, but the number 1 is. If you want the previous index to be set, be aware to check individually for its existence as the current if will not set the index as `RENT` if the other one doesn't exist.
You can use `getAssetIndexesUpdatesHistoryCount` to get the current index.

**Review by LautaroPetaccio — CHANGES_REQUESTED** [review]

File: src/handlers/rentals.ts (hunk 3 above)
Comment (LautaroPetaccio) [reply src/handlers/rentals.ts]: After re-reviewing the code, you were right. The `getIndexesUpdatesHistoryCount` is the function to use. The other indexes are related to the entity type that is described in their parent entity, the index history entity.

**Review by LautaroPetaccio — APPROVED** [review]

**Pull request merged**